### PR TITLE
Add basic UI

### PR DESCRIPTION
**Pull request by MattGomb**
Dear Reviewer!

In this PR I implemented the required changes for the project [OOP school library: add basic UI](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_ui.md):

- Created the `app.rb` and `main.rb`
- created the interface in `main.rb`
- created functionality in `app.rb`
- tested every function, they work
- edge cases and typing mistakes are not protected against, but they were not shown in the video either.

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+      print 'ID of person: '
+      selected_id = gets.chomp.to_i
+      Rental.all.each do |rental|
+        if rental.person.id == selected_id
+          puts %(Date: #{rental.date}, Book: #{rental.book.title} by #{rental.book.author})
+        else
+          puts 'No rentals found for that ID!'
+        end
+      end
+    end
```
Comment (BenMukebo) [inline app.rb:110]: - [OPTIONAL] Great job so far 👏🏻From Steps one up to five works fine, however, the last one step six has some mistakes that need to be addressed. Here is the screenshot 👇🏻 Please Remove the `No rentals found for that ID!` message. this must only display when there are not any rentals created yet!!


Screenshot from 2023-01-13 04-50-03

File: app.rb [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+  end
+
+  def create_book
+    print 'Title: '
+    title = gets.chomp
+    print 'Author: '
+    author = gets.chomp
+    Book.new(title, author)
+    puts 'Book added to the database successfully!'
+  end
```
Comment (BenMukebo) [inline app.rb:71]: - [OPTIONAL] Make sure books are created in ascending order. This means the first book should have the first id and the last should have the highest book id

Screenshot from 2023-01-13 04-54-40

**Review by BenMukebo — APPROVED** [review]
<h1>Status: Approve  :champagne:  :+1: :confetti_ball: 

### **Hi @MattGomb**, 

### **Congratulations! 🎉 Project Approved ✅**

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 


### To Highlight :1st_place_medal: 
- Your Pull Request has a proper title and description. ✔️  
- No Linter errors :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers :clinking_glasses:  and Happy coding!!! :dancers:💻  

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @BenMukebo when you comment so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app.rb (hunk 1 above)
Comment (MattGomb) [reply app.rb:110]: Thank you i did not notice that! will fix!

File: app.rb (hunk 2 above)
Comment (MattGomb) [reply app.rb:71]: thank you will fix that too!

**Pull request merged**